pkgs/by-name/od/odin/package.nix

```diff
@@ -60,6 +66,11 @@ stdenv.mkDerivation {
     mkdir -p $out/share
     cp -r {base,core,vendor,shared} $out/share
 
+    ${lib.optionalString useWgpu ''
+      mkdir $out/share/vendor/wgpu/lib/wgpu-linux-x86_64-release/
```

Comment:
This path should not be system specific (adjust it according to `stdenv.hostPlatform`).